### PR TITLE
InfluxDB 2.0.3 changelog

### PR DESCRIPTION
**Pull request by noramullen1**

### CODE REVIEW

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/influxdb.md:38]: ...index matches exact foreign keys only?

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (timhallinflux) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:26]: Also... are we adding the deb/rpm packaging too? @psteinbachs

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (timhallinflux) [inline content/influxdb/v2.0/reference/release-notes/influxdb.md:16]: `config,toml` was written to the `home` directory of the user who launched the upgrade.  (this describes the previous behavior).

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (timhallinflux) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:29]: agreed

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (timhallinflux) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:35]: agreed

**Review by timhallinflux — CHANGES_REQUESTED** [review]

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (danxmoran) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:37]: We were hitting an edge case where an array started at 100, and we'd double it in size as needed _except_ when there were exactly as many tags as the size of the array. The reporter found it for 100 tags, but it was probably 100, 200, 400, 800, ... etc.

Not sure of the most concise way to say that :smile:

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (danxmoran) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:38]: If we want a more user-friendly description, we could mention the specific error that led Stuart to make this fix, something like:
```
Fix validation of existing DB names when creating DBRP mappings
```

**Comment by danxmoran** [comment]
@noramullen1 @kelseiv this issue was also fixed by a Flux upgrade, I didn't notice it in the notes here: https://github.com/influxdata/influxdb/issues/20308

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (noramullen1) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:26]: phil pinged me to confirm they will be adding this, he's just waiting on final review of his PR

**Comment by noramullen1** [comment]
> (

Thanks @danxmoran, added.

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (noramullen1) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:33]: added @timhallinflux

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/influxdb.md:20]: need space after line 20 to create bullet list

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/influxdb.md:38]: to [Flux v0.98.0](/influxdb/v2.0/reference/release-notes/flux/#v0-98-0-2020-12-07).

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (kelseiv) [inline content/influxdb/v2.0/reference/release-notes/influxdb.md:41]: extra sp

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.0/reference/release-notes/influxdb.md
Comment (danxmoran) [reply content/influxdb/v2.0/reference/release-notes/influxdb.md:38]: Changed as of this afternoon: we're now at Flux v0.99.0 and flux-lsp-browser v0.5.26

File: content/influxdb/cloud/get-started.md
Comment (danxmoran) [inline content/influxdb/cloud/get-started.md:93]: Here and elsewhere, it looks like our artifact names have slightly changed. Instead of an `influxdb` prefix, we use `influxdb2`. So this would be:
```suggestion
<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2_client_2.0.3_darwin_amd64.tar.gz" download>influx CLI (macOS)</a>
```
cc @psteinbachs

Comment (noramullen1) [reply content/influxdb/cloud/get-started.md]: Fixed

**Pull request merged**
